src/buffer.hh

```diff
@@ -249,14 +252,17 @@ private:
         StringView back() const { return BufferLines::back()->strview(); }
     };
     LineList m_lines;
+    ByteOrderMark m_byte_order_mark = ByteOrderMark::None;
+    EolFormat m_eol_format = EolFormat::Lf;
 
     String m_name;
     String m_display_name;
     Flags  m_flags;
+    ByteCount m_contents_size;
```

Comment:
Same comment, I am not keen on storing that as well, why would we want to keep around the buffer size at last reload ? We can compute it if needed.

Reply:
Getting the file size requires issuing an expensive syscall, we can simply store the integer since it's computed right before every time we need it.

Reply:
Which syscall ? my understanding is that this is the size of the file at last load. Is that correct to compare to size at last load instead of current size ? In that case why are we comparing the current content of the buffer instead of the last loaded content ?

Reply:
Getting a file's size is generally done using `stat()`.

Saving the file size only on reload is probably not a good idea, as it will be different from the size of the buffer on a simple `:write`.